yarn.lock

```diff
@@ -1486,18 +1486,20 @@ tslint@^5.4.2:
     resolve "^1.3.2"
     semver "^5.3.0"
     tslib "^1.7.1"
-    tsutils "^2.3.0"
-
-tsutils@^2.3.0:
-  version "2.4.0"
-  resolved "https://registry.yarnpkg.com/tsutils/-/tsutils-2.4.0.tgz#ad4ce6dba0e5a3edbddf8626b7ca040782189fea"
+    tsutils "^2.5.1"
 
 tsutils@^2.5.1:
```

Comment:
we could merge this block from the lockfile with the block below for `2.6.0` satisfies the requirement for both